website/blog/2021/09/09/how-to-contribute-to-an-OpenSource-without-coding.md

```diff
@@ -1,108 +1,108 @@
 ---
-title: "不写代码，如何为开源项目做贡献？"
-author: "赵若妃"
+title: How can I contribute to an open source project without writing code?
+author: Ruofei Zhao
 authorURL: "https://github.com/Serendipity96"
 authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
 keywords: 
-- 开源项目
-- 贡献
-description: 本文介绍了很多不写代码为开源项目贡献的方式，可以通过写文章、做视频、对外分享、建立本地社区、积极参与社区讨论等方式参与到开源项目中。
+- Open Source
+- Contribution
+description: This article introduces many ways to contribute to open source projects without writing code. You can participate in open source projects by writing articles, making videos, sharing them externally, building local communities, and actively participating in community discussions.
 tags: [Community]
 ---
 
-> 本文介绍了很多不写代码为开源项目贡献的方式，可以通过写文章、做视频、对外分享、建立本地社区、积极参与社区讨论等方式参与到开源项目中。
+> This article describes many ways to contribute to open source projects without writing code. You can get involved in open source projects by writing articles, making videos, sharing them externally, building local communities, and participating actively in community discussions.
 
-<!--truncate-->
+<! --truncate -->
 
-笔者为开源项目贡献过几个 PR，提交过几个 issue，直到现在还清晰记得几年前第一个 PR 被 merge 时的兴奋，体验到了参与开源的乐趣。但是只停留在了 contributor 阶段，没有下文了，因为笔者后来不做研发了，**认为不熟悉代码、不能修 bug 就没有贡献的机会了。**
+I have contributed several PRs and submitted several issues to open source projects, and I still remember clearly the excitement when my first PR was merged a few years ago, and I experienced the fun of participating in open source. But only in the contributor stage, no more, because I later do not do R & D, **think not familiar with the code, can not fix the bug will not have the opportunity to contribute.**
```

Comment:
```suggestion
I have contributed several PRs and submitted several issues to open source projects, and I still remember clearly the excitement when my first PR was merged a few years ago, and I experienced the fun of participating in open source. But my experience remains in the contributor stage with no further progress, because I changed my position from front-end developer to marketing. I am afraid that I would be less familiar with the code,  and I can not fix the bug, which will result in no contribution to the project.
```